app/controllers/patients_controller.rb

```diff
@@ -54,26 +55,36 @@ def find_patient
     @patient = Patient.find params[:id]
   end
 
+  # Strong params divided up by partial, in order of appearance
```

Comment:
yeah, in order of appearance where. I meant to say 'appearance in their respective partials', but I think just partial is fine. Removing everything after the comma here for simplicity.